> please fix CI


```
[INFO] 
[ERROR] Failures: 
[ERROR]   SuppressWarningsHolderExamplesTest.testExample1:51->AbstractModuleTestSupport.verifyWithInlineConfigParser:297->AbstractModuleTestSupport.verifyViolations:754 Violation lines for /home/vsts/work/1/s/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example1.java differ.
missing (1): 34
---
expected   : [20, 28, 28, 34]
but was    : [20, 28, 28]
[INFO] 
[ERROR] Tests run: 6067, Failures: 1, Errors: 0, Skipped: 2
[INFO] 
```

I don't see this error locally, I dont understand why this is failing here when this violation is expected.
Using both ParamNumberId and paramnum in one example is causing this to fail on ci, locally this is passing

--- a/config/jsoref-spellchecker/whitelist.words
+++ b/config/jsoref-spellchecker/whitelist.words
@@ -1050,10 +1050,10 @@ pageview
 Paikin
 Papaioannou
 paraception
+paramcounter
 parameterassignment
 parametername
 parameternumber
-paramnum
 Parcelable
 parenpad
 PARM

--- a/src/site/xdoc/checks/annotation/suppresswarningsholder.xml
+++ b/src/site/xdoc/checks/annotation/suppresswarningsholder.xml
@@ -50,7 +50,7 @@
     &lt;module name="MemberName"/&gt;
     &lt;module name="ConstantName"/&gt;
     &lt;module name="ParameterNumber"&gt;
-      &lt;property name="id" value="ParamNumberId"/&gt;
+      &lt;property name="id" value="ParamListShouldBeShort"/&gt;
     &lt;/module&gt;
     &lt;module name="NoWhitespaceAfter"/&gt;
     &lt;module name="SuppressWarningsHolder"/&gt;
@@ -65,20 +65,18 @@ class Example1 {
   @SuppressWarnings({"membername"})
   private int J; // violation suppressed
 
-  private static final int i = 0; // violation, 'Name 'i' must match pattern'
-  @SuppressWarnings("checkstyle:constantname")
-  private static final int m = 0; // violation suppressed
-
-  @SuppressWarnings("ParamNumberId")
+  @SuppressWarnings("ParamListShouldBeShort")
     public void needsLotsOfParameters1 (int a, // violation suppressed
-     int b, int c, int d, int e, int f, int g, int h) {
-     // ...
-  }
+     int b, int c, int d, int e, int f, int g, int h) {}
 
   private int [] ARR; // violation ''int' is followed by whitespace'
   // violation above, 'Name 'ARR' must match pattern'
   @SuppressWarnings("all")
   private int [] ARRAY; // violation suppressed
+  // violation 2 lines below 'More than 7 parameters (found 8)'
+  @SuppressWarnings("paramcounter")
+    public void needsLotsOfParameters2 (int a,
+     int b, int c, int d, int e, int f, int g, int h) {}
 }
 </code></pre></div><hr class="example-separator"/>
         <p>The general rule is that the argument of the <code>@SuppressWarnings</code> will be
@@ -88,40 +86,51 @@ class Example1 {
         <p id="Example2-config">
           If <code>aliasList</code> property was provided you can use your own names e.g. below
           code will work if there was provided a
-          <code>com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramnum</code>
+          <code>
+          com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramcounter</code>
           in the <code>aliasList</code>:
         </p>
         <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
 &lt;module name="Checker"&gt;
   &lt;module name="TreeWalker"&gt;
-    &lt;module name="ParameterNumber"/&gt;
+    &lt;module name="MemberName"/&gt;
+    &lt;module name="ConstantName"/&gt;
+    &lt;module name="ParameterNumber"&gt;
+      &lt;property name="id" value="ParamListShouldBeShort"/&gt;
+    &lt;/module&gt;
     &lt;module name="SuppressWarningsHolder"&gt;
       &lt;property name="aliasList"
-        value="com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramnum"/&gt;
+        value="com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramcounter"/&gt;
     &lt;/module&gt;
+    &lt;module name="NoWhitespaceAfter"/&gt;
+    &lt;module name="SuppressWarningsHolder"/&gt;
   &lt;/module&gt;
   &lt;module name="SuppressWarningsFilter"/&gt;
 &lt;/module&gt;
 </code></pre></div>
         <p id="Example2-code">Example:</p>
         <div class="wrapper"><pre class="prettyprint"><code class="language-java">
-public class Example2 {
-  // violation below, 'More than 7 parameters (found 8)'
-  public void needsLotsOfParameters (int a,
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+class Example2 {
+  private int K; // violation, 'Name 'K' must match pattern'
+  @SuppressWarnings({"membername"})
+  private int J; // violation suppressed
 
-  @SuppressWarnings("paramnum")
-  public void needsLotsOfParameters1 (int a, // violation suppressed
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  @SuppressWarnings("ParamListShouldBeShort")
+  public void needsLotsOfParameters1 (int a,
+    int b, int c, int d, int e, int f, int g, int h) {}
 
+  private int [] ARR; // violation ''int' is followed by whitespace'
+  // violation above, 'Name 'ARR' must match pattern'
+  @SuppressWarnings("all")
+  private int [] ARRAY; // violation suppressed
+
+  @SuppressWarnings("paramcounter")
+  public void needsLotsOfParameters2 (int a, // violation suppressed
+    int b, int c, int d, int e, int f, int g, int h) {}
 }
 </code></pre></div><hr class="example-separator"/>
         <p id="Example3-config">
-          You can also use simple check name like <code>ParameterNumberCheck=paramnum</code>
+          You can also use simple check name like <code>ParameterNumberCheck=paramcounter</code>
           instead to use fully qualified name of check in the <code>aliasList</code>:
         </p>
         <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
@@ -129,7 +138,7 @@ public class Example2 {
   &lt;module name="TreeWalker"&gt;
     &lt;module name="ParameterNumber"/&gt;
     &lt;module name="SuppressWarningsHolder"&gt;
-      &lt;property name="aliasList" value="ParameterNumberCheck=paramnum"/&gt;
+      &lt;property name="aliasList" value="ParameterNumberCheck=paramcounter"/&gt;
     &lt;/module&gt;
   &lt;/module&gt;
   &lt;module name="SuppressWarningsFilter"/&gt;
@@ -138,30 +147,49 @@ public class Example2 {
         <p id="Example3-code">Example:</p>
         <div class="wrapper"><pre class="prettyprint"><code class="language-java">
 public class Example3 {
+
+  private int K;
+
+  @SuppressWarnings({"membername"})
+  private int J;
+
+  private static final int i = 0;
+
+  @SuppressWarnings("checkstyle:constantname")
+  private static final int m = 0;
+
   // violation below, 'More than 7 parameters (found 8)'
-  public void needsLotsOfParameters (int a,
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  public void needsLotsOfParameters(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {}
 
-  @SuppressWarnings("paramnum")
-  public void needsLotsOfParameters1 (int a, // violation suppressed
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  // violation 2 lines below 'More than 7 parameters (found 8)'
+  @SuppressWarnings("ParamNumberId")
+  public void needsLotsOfParameters1(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {}
 
+  @SuppressWarnings("paramcounter")
+  public void needsLotsOfParameters2(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {} // violation suppressed
+
+  private int [] ARR;
+
+  @SuppressWarnings("all")
+  private int [] ARRAY;
 }
 </code></pre></div><hr class="example-separator"/>
         <p id="Example4-config">
           The check can also be used without suffix of &quot;Check&quot; like
-          <code>ParameterNumber=paramnum</code> in the <code>aliasList</code>:
+          <code>ParameterNumber=paramcounter</code> in the <code>aliasList</code>:
         </p>
         <div class="wrapper"><pre class="prettyprint"><code class="language-xml">
 &lt;module name="Checker"&gt;
   &lt;module name="TreeWalker"&gt;
     &lt;module name="ParameterNumber"/&gt;
     &lt;module name="SuppressWarningsHolder"&gt;
-      &lt;property name="aliasList" value="ParameterNumber=paramnum"/&gt;
+      &lt;property name="aliasList" value="ParameterNumber=paramcounter"/&gt;
     &lt;/module&gt;
   &lt;/module&gt;
   &lt;module name="SuppressWarningsFilter"/&gt;
@@ -170,18 +198,37 @@ public class Example3 {
         <p id="Example4-code">Example:</p>
         <div class="wrapper"><pre class="prettyprint"><code class="language-java">
 public class Example4 {
+
+  private int K;
+
+  @SuppressWarnings({"membername"})
+  private int J;
+
+  private static final int i = 0;
+
+  @SuppressWarnings("checkstyle:constantname")
+  private static final int m = 0;
+
   // violation below, 'More than 7 parameters (found 8)'
-  public void needsLotsOfParameters (int a,
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  public void needsLotsOfParameters(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {}
 
-  @SuppressWarnings("paramnum")
-  public void needsLotsOfParameters1 (int a, // violation suppressed
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  // violation 2 lines below 'More than 7 parameters (found 8)'
+  @SuppressWarnings("ParamNumberId")
+  public void needsLotsOfParameters1(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {}
 
+  @SuppressWarnings("paramcounter")
+  public void needsLotsOfParameters2(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {} // violation suppressed
+
+  private int [] ARR;
+
+  @SuppressWarnings("all")
+  private int [] ARRAY;
 }
 </code></pre></div>
 

--- a/src/site/xdoc/checks/annotation/suppresswarningsholder.xml.template
+++ b/src/site/xdoc/checks/annotation/suppresswarningsholder.xml.template
@@ -48,7 +48,8 @@
         <p id="Example2-config">
           If <code>aliasList</code> property was provided you can use your own names e.g. below
           code will work if there was provided a
-          <code>com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramnum</code>
+          <code>
+          com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramcounter</code>
           in the <code>aliasList</code>:
         </p>
         <macro name="example">
@@ -63,7 +64,7 @@
           <param name="type" value="code"/>
         </macro><hr class="example-separator"/>
         <p id="Example3-config">
-          You can also use simple check name like <code>ParameterNumberCheck=paramnum</code>
+          You can also use simple check name like <code>ParameterNumberCheck=paramcounter</code>
           instead to use fully qualified name of check in the <code>aliasList</code>:
         </p>
         <macro name="example">
@@ -79,7 +80,7 @@
         </macro><hr class="example-separator"/>
         <p id="Example4-config">
           The check can also be used without suffix of &quot;Check&quot; like
-          <code>ParameterNumber=paramnum</code> in the <code>aliasList</code>:
+          <code>ParameterNumber=paramcounter</code> in the <code>aliasList</code>:
         </p>
         <macro name="example">
           <param name="path"

--- a/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsExamplesAstConsistencyTest.java
+++ b/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsExamplesAstConsistencyTest.java
@@ -102,9 +102,6 @@ public class XdocsExamplesAstConsistencyTest {
      */
     private static final Set<String> SUPPRESSED_EXAMPLES = Set.of(
             "checks/annotation/suppresswarnings/Example2",
-            "checks/annotation/suppresswarningsholder/Example2",
-            "checks/annotation/suppresswarningsholder/Example3",
-            "checks/annotation/suppresswarningsholder/Example4",
             "checks/blocks/leftcurly/Example3",
             "checks/coding/equalsavoidnull/Example2",
             "checks/coding/explicitinitialization/Example2",
@@ -300,6 +297,8 @@ public class XdocsExamplesAstConsistencyTest {
             "checks/regexp/regexpsingleline/Example5",
             "checks/javadoc/writetag/Example5",
             "checks/coding/illegalinstantiation/Example3",
+            "checks/annotation/suppresswarningsholder/Example3",
+            "checks/annotation/suppresswarningsholder/Example4",
             "checks/coding/matchxpath/Example2",
             "checks/coding/matchxpath/Example3",
             "checks/coding/matchxpath/Example4",

--- a/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/annotation/SuppressWarningsHolderExamplesTest.java
+++ b/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/annotation/SuppressWarningsHolderExamplesTest.java
@@ -23,7 +23,6 @@ import org.junit.jupiter.api.Test;
 
 import com.puppycrawl.tools.checkstyle.AbstractExamplesModuleTestSupport;
 import com.puppycrawl.tools.checkstyle.checks.naming.AbstractNameCheck;
-import com.puppycrawl.tools.checkstyle.checks.naming.ConstantNameCheck;
 import com.puppycrawl.tools.checkstyle.checks.naming.MemberNameCheck;
 import com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck;
 import com.puppycrawl.tools.checkstyle.checks.whitespace.NoWhitespaceAfterCheck;
@@ -37,16 +36,15 @@ public class SuppressWarningsHolderExamplesTest extends AbstractExamplesModuleTe
     @Test
     public void testExample1() throws Exception {
         final String pattern1 = "^[a-z][a-zA-Z0-9]*$";
-        final String pattern2 = "^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$";
         final String[] expected = {
             "20:15: " + getCheckMessage(MemberNameCheck.class,
                         AbstractNameCheck.MSG_INVALID_PATTERN, "K", pattern1),
-            "24:28: " + getCheckMessage(ConstantNameCheck.class,
-                        AbstractNameCheck.MSG_INVALID_PATTERN, "i", pattern2),
-            "34:15: " + getCheckMessage(NoWhitespaceAfterCheck.class,
+            "28:15: " + getCheckMessage(NoWhitespaceAfterCheck.class,
                         NoWhitespaceAfterCheck.MSG_KEY, "int"),
-            "34:18: " + getCheckMessage(MemberNameCheck.class,
+            "28:18: " + getCheckMessage(MemberNameCheck.class,
                         AbstractNameCheck.MSG_INVALID_PATTERN, "ARR", pattern1),
+            "34:17: " + getCheckMessage(ParameterNumberCheck.class,
+                        ParameterNumberCheck.MSG_KEY, 7, 8),
 
         };
 
@@ -55,10 +53,14 @@ public class SuppressWarningsHolderExamplesTest extends AbstractExamplesModuleTe
 
     @Test
     public void testExample2() throws Exception {
+        final String pattern1 = "^[a-z][a-zA-Z0-9]*$";
         final String[] expected = {
-            "19:15: " + getCheckMessage(ParameterNumberCheck.class,
-                        ParameterNumberCheck.MSG_KEY, 7, 8),
-
+            "24:15: " + getCheckMessage(MemberNameCheck.class,
+                        AbstractNameCheck.MSG_INVALID_PATTERN, "K", pattern1),
+            "32:15: " + getCheckMessage(NoWhitespaceAfterCheck.class,
+                        NoWhitespaceAfterCheck.MSG_KEY, "int"),
+            "32:18: " + getCheckMessage(MemberNameCheck.class,
+                        AbstractNameCheck.MSG_INVALID_PATTERN, "ARR", pattern1),
         };
 
         verifyWithInlineConfigParser(getPath("Example2.java"), expected);
@@ -67,7 +69,9 @@ public class SuppressWarningsHolderExamplesTest extends AbstractExamplesModuleTe
     @Test
     public void testExample3() throws Exception {
         final String[] expected = {
-            "18:15: " + getCheckMessage(ParameterNumberCheck.class,
+            "29:15: " + getCheckMessage(ParameterNumberCheck.class,
+                        ParameterNumberCheck.MSG_KEY, 7, 8),
+            "35:15: " + getCheckMessage(ParameterNumberCheck.class,
                         ParameterNumberCheck.MSG_KEY, 7, 8),
         };
 
@@ -77,7 +81,9 @@ public class SuppressWarningsHolderExamplesTest extends AbstractExamplesModuleTe
     @Test
     public void testExample4() throws Exception {
         final String[] expected = {
-            "18:15: " + getCheckMessage(ParameterNumberCheck.class,
+            "29:15: " + getCheckMessage(ParameterNumberCheck.class,
+                        ParameterNumberCheck.MSG_KEY, 7, 8),
+            "35:15: " + getCheckMessage(ParameterNumberCheck.class,
                         ParameterNumberCheck.MSG_KEY, 7, 8),
         };
 

--- a/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example1.java
+++ b/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example1.java
@@ -4,7 +4,7 @@
     <module name="MemberName"/>
     <module name="ConstantName"/>
     <module name="ParameterNumber">
-      <property name="id" value="ParamNumberId"/>
+      <property name="id" value="ParamListShouldBeShort"/>
     </module>
     <module name="NoWhitespaceAfter"/>
     <module name="SuppressWarningsHolder"/>
@@ -21,19 +21,17 @@ class Example1 {
   @SuppressWarnings({"membername"})
   private int J; // violation suppressed
 
-  private static final int i = 0; // violation, 'Name 'i' must match pattern'
-  @SuppressWarnings("checkstyle:constantname")
-  private static final int m = 0; // violation suppressed
-
-  @SuppressWarnings("ParamNumberId")
+  @SuppressWarnings("ParamListShouldBeShort")
     public void needsLotsOfParameters1 (int a, // violation suppressed
-     int b, int c, int d, int e, int f, int g, int h) {
-     // ...
-  }
+     int b, int c, int d, int e, int f, int g, int h) {}
 
   private int [] ARR; // violation ''int' is followed by whitespace'
   // violation above, 'Name 'ARR' must match pattern'
   @SuppressWarnings("all")
   private int [] ARRAY; // violation suppressed
+  // violation 2 lines below 'More than 7 parameters (found 8)'
+  @SuppressWarnings("paramcounter")
+    public void needsLotsOfParameters2 (int a,
+     int b, int c, int d, int e, int f, int g, int h) {}
 }
 // xdoc section -- end

--- a/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example2.java
+++ b/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example2.java
@@ -1,11 +1,17 @@
 /*xml
 <module name="Checker">
   <module name="TreeWalker">
-    <module name="ParameterNumber"/>
+    <module name="MemberName"/>
+    <module name="ConstantName"/>
+    <module name="ParameterNumber">
+      <property name="id" value="ParamListShouldBeShort"/>
+    </module>
     <module name="SuppressWarningsHolder">
       <property name="aliasList"
-        value="com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramnum"/>
+        value="com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramcounter"/>
     </module>
+    <module name="NoWhitespaceAfter"/>
+    <module name="SuppressWarningsHolder"/>
   </module>
   <module name="SuppressWarningsFilter"/>
 </module>
@@ -14,18 +20,22 @@
 package com.puppycrawl.tools.checkstyle.checks.annotation.suppresswarningsholder;
 
 // xdoc section -- start
-public class Example2 {
-  // violation below, 'More than 7 parameters (found 8)'
-  public void needsLotsOfParameters (int a,
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+class Example2 {
+  private int K; // violation, 'Name 'K' must match pattern'
+  @SuppressWarnings({"membername"})
+  private int J; // violation suppressed
 
-  @SuppressWarnings("paramnum")
-  public void needsLotsOfParameters1 (int a, // violation suppressed
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  @SuppressWarnings("ParamListShouldBeShort")
+  public void needsLotsOfParameters1 (int a,
+    int b, int c, int d, int e, int f, int g, int h) {}
 
+  private int [] ARR; // violation ''int' is followed by whitespace'
+  // violation above, 'Name 'ARR' must match pattern'
+  @SuppressWarnings("all")
+  private int [] ARRAY; // violation suppressed
+
+  @SuppressWarnings("paramcounter")
+  public void needsLotsOfParameters2 (int a, // violation suppressed
+    int b, int c, int d, int e, int f, int g, int h) {}
 }
 // xdoc section -- end

--- a/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example3.java
+++ b/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example3.java
@@ -3,7 +3,7 @@
   <module name="TreeWalker">
     <module name="ParameterNumber"/>
     <module name="SuppressWarningsHolder">
-      <property name="aliasList" value="ParameterNumberCheck=paramnum"/>
+      <property name="aliasList" value="ParameterNumberCheck=paramcounter"/>
     </module>
   </module>
   <module name="SuppressWarningsFilter"/>
@@ -14,17 +14,37 @@ package com.puppycrawl.tools.checkstyle.checks.annotation.suppresswarningsholder
 
 // xdoc section -- start
 public class Example3 {
+
+  private int K;
+
+  @SuppressWarnings({"membername"})
+  private int J;
+
+  private static final int i = 0;
+
+  @SuppressWarnings("checkstyle:constantname")
+  private static final int m = 0;
+
   // violation below, 'More than 7 parameters (found 8)'
-  public void needsLotsOfParameters (int a,
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  public void needsLotsOfParameters(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {}
 
-  @SuppressWarnings("paramnum")
-  public void needsLotsOfParameters1 (int a, // violation suppressed
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  // violation 2 lines below 'More than 7 parameters (found 8)'
+  @SuppressWarnings("ParamNumberId")
+  public void needsLotsOfParameters1(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {}
 
+  @SuppressWarnings("paramcounter")
+  public void needsLotsOfParameters2(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {} // violation suppressed
+
+  private int [] ARR;
+
+  @SuppressWarnings("all")
+  private int [] ARRAY;
 }
 // xdoc section -- end
+

--- a/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example4.java
+++ b/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example4.java
@@ -3,7 +3,7 @@
   <module name="TreeWalker">
     <module name="ParameterNumber"/>
     <module name="SuppressWarningsHolder">
-      <property name="aliasList" value="ParameterNumber=paramnum"/>
+      <property name="aliasList" value="ParameterNumber=paramcounter"/>
     </module>
   </module>
   <module name="SuppressWarningsFilter"/>
@@ -14,17 +14,36 @@ package com.puppycrawl.tools.checkstyle.checks.annotation.suppresswarningsholder
 
 // xdoc section -- start
 public class Example4 {
+
+  private int K;
+
+  @SuppressWarnings({"membername"})
+  private int J;
+
+  private static final int i = 0;
+
+  @SuppressWarnings("checkstyle:constantname")
+  private static final int m = 0;
+
   // violation below, 'More than 7 parameters (found 8)'
-  public void needsLotsOfParameters (int a,
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  public void needsLotsOfParameters(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {}
 
-  @SuppressWarnings("paramnum")
-  public void needsLotsOfParameters1 (int a, // violation suppressed
-    int b, int c, int d, int e, int f, int g, int h) {
-    // ...
-  }
+  // violation 2 lines below 'More than 7 parameters (found 8)'
+  @SuppressWarnings("ParamNumberId")
+  public void needsLotsOfParameters1(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {}
 
+  @SuppressWarnings("paramcounter")
+  public void needsLotsOfParameters2(
+          int a, int b, int c, int d,
+          int e, int f, int g, int h) {} // violation suppressed
+
+  private int [] ARR;
+
+  @SuppressWarnings("all")
+  private int [] ARRAY;
 }
 // xdoc section -- end

